app/queries/my_facilities/blood_pressure_control_query.rb

```diff
@@ -53,24 +53,27 @@ def cohort_missed_visits_count_by_facility
 
   def overall_patients
     @overall_patients ||= Patient
-                          .where('recorded_at < ?', Time.current.beginning_of_day - REGISTRATION_BUFFER)
-                          .where(registration_facility: facilities)
+                            .hypertension_only
+                            .where('recorded_at < ?', Time.current.beginning_of_day - REGISTRATION_BUFFER)
+                            .where(registration_facility: facilities)
   end
 
   def overall_controlled_bps
     @overall_controlled_bps ||=
       LatestBloodPressuresPerPatient
-      .where(patient: overall_patients)
-      .where('bp_recorded_at > ?', Time.current.beginning_of_day - 90.days)
-      .under_control
+        .where(patient: overall_patients)
+        .where('bp_recorded_at > ?', Time.current.beginning_of_day - 90.days)
+        .under_control
   end
 
   private
 
   attr_reader :facilities
 
   def quarterly_registrations
-    patients = Patient.where(registration_facility: facilities)
+    patients = Patient
+                 .hypertension_only
```

Comment:
I love how easy this is. As mentioned in my review on #897, I'd love for this scope to be named something like `with_hypertension` or `diagnosed_with_hypertension`, since `hypertension_only` makes it seem like patients with both HTN and DM are excluded from this scope (but they're not).